fcrepo-clients/src/main/java/edu/unc/lib/dl/util/VocabularyHelperManager.java

```diff
@@ -86,6 +86,9 @@
 	private Boolean initialized = false;
 
 	public synchronized void init() {
+		// Wait for the repository to be up before loading vocabularies
+		managementClient.waitForRepositoryAvailable();
```

Comment:
Since other parts of the deposit service depend on the availability of the repository, could this be moved into DepositSupervisor?